packages/pwa-kit-react-sdk/src/utils/performance.test.js

```diff
@@ -42,4 +53,96 @@ describe('PerformanceTimer', () => {
         expect(timer.metrics[0].name).toBe('test')
         expect(parseFloat(timer.metrics[0].duration)).toBeGreaterThan(0)
     })
+
+    test('warns when mark name is undefined', () => {
```

Comment:
To meet the test coverage, I decided to add tests for this performance util. Yes, it's not related to the PR. But there isn't much changes in the react-rendering.js file that's worthy for testing.